src/templates/english/index.js

```diff
@@ -0,0 +1,24 @@
+import email from "./Email.svelte";
+
+export const english = {
+  templates: {
+    email: {
+      label: "Email",
+      value: "email",
+      template: email,
+      isComponent: true,
+    },
+    twitter: {
+      label: "Twitter",
+      value: "twitter",
+      template: "hi...",
+    },
+    linkedin: {
+      label: "LinkedIn",
+      value: "linkedin",
+      template: "LinkedIn template",
+    },
+  },
+  label: "English",
+  value: "english",
+};
```

Comment:
I've added a couple templates, for English and Russian. Some of the are just placeholders for now, I'll add some real text later.